db/migrate/20170326222205_add_time_spent_to_library_entry.rb

```diff
@@ -0,0 +1,23 @@
+require 'update_in_batches'
+
+class AddTimeSpentToLibraryEntry < ActiveRecord::Migration
+  using UpdateInBatches
+
+  self.disable_ddl_transaction!
+
+  def up
+    add_column :library_entries, :time_spent, :integer
+
+    say_with_time 'Filling time_spent column' do
+      LibraryEntry.where(media_type: 'Anime').joins(:anime)
```

Comment:
Nonono, replace `where(media_type: 'Anime')` with `by_kind(:anime)`

As the name implies, `by_kind` gets Library Entries _by kind_ 😉